<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabResultInsertion</class>
 <widget class="QWidget" name="LabResultInsertion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Atr√°s</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar resultado de laboratorio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="isPending">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>92</width>
     <height>24</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pendiente</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="formTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>331</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkPatientData">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Revisar Datos de Paciente</string>
   </property>
  </widget>
  <widget class="QLabel" name="formLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Formulario</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
